.gitmodules

```diff
@@ -1,3 +1,3 @@
 [submodule "azure-functions-shared/protobuf"]
     path = azure-functions-shared/protobuf
-    url = https://github.com/Azure/azure-functions-language-worker-protobuf
+    url = https://github.com/peterhuene/azure-functions-language-worker-protobuf
```

Comment:
Why are you referencing your fork? Is this because you need to use the `fix-package` branch?

Reply:
That's correct.  `prost` needs a package declaration in the file (they closed an issue that pointed out that it's an optional directive), which is missing from two of the three proto files.

I'll see if they can take this fix upstream or add a package declaration that makes sense for them.

In the short term we can use my fork since it's up-to-date.